app/scripts/geocoder/geocoder-service.js

```diff
@@ -18,7 +18,7 @@
         var searchOutFields = 'StAddr,City,Postal';
         var searchCategories = [
             'Street Address',
```

Comment:
Oh. Very interesting. I just tested `1808 spring garden st` and it properly found it with whats set here. But I'll add postal anyways just in case and we can adjust later.